File: pset1/mario/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int h; //height
    int i; //row
    int j; //column

    //prompt user for height until integer value of [1,8] is provided
    do
    {
        h = get_int("Height: ");
    }
    while (h < 1 || h > 8);

    //Iterate until there are h number of rows
    for (i = 1; i <= h ; i++)
    {
        //Iterate until there are h number of columns within a row
        for (j = 1; j <= h; j++)
        {
            if (i + j <= h) //Print spaces if coordinate is less than height
            {
                printf(" ");
            }
            else //Print hashes if coordinate points is > height
            {
                printf("#");
            }
        }
        printf("\n");
    }
}

File: pset6/mario.py
from cs50 import get_int

# request height until it's [1,8]
height = 0
while (height < 1 or height > 8):
    height = get_int("Height: ")

# for each line, print the calculated amount of spaces and #. Number of # is equivalent to the row number (which begins with 0 b/c of range). Spaces is equivalent to remainder.
for row in range(height):
    print(" "*(height - row - 1) + "#"*(row + 1))

